docs/topics/using/edgectl/service-preview-install.md

```diff
@@ -2,42 +2,231 @@
-
-- [`edgectl install`](#install-with-edgectl) will boot strap your cluster with the Ambassador Edge Stack and Service Preview and only works when doing a fresh install of Ambassador.
-
-- [`YAML Install`](#install-with-yaml) will work with an existing installation of Ambassador Edge Stack.
-
 ## Prerequisites
 
 - [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) access to a Kubernetes cluster
-- The Ambassador Edge Stack [Installed](../../../../tutorials/getting-started/)
 - [`edgectl`](../edge-control#installing-edge-control) client
 
-## Install with Edgectl
+## Install
+
+There are three method for installing Service Preview.
+
+### [<img class="Ambassador's OpenSource Blackbird" src="../../../images/features-page-bird.svg"/> Install with Edgectl](#install-with-edgectl)
+
+If you are installing Service Preview and Ambassador Edge Stack for the first time, `edgectl` will bootstrap your cluster with the Ambassador Edge Stack and Service Preview automatically.
+
+### [<img class="k8s-logo" src="../../../images/kubernetes.png"/> Install with YAML](install with yaml)
+
+The YAML installation method will walk you through a step-by-step deployment of all the resources necessary for installing Service Preview alongside the Ambassador Edge Stack. The YAML installation method is the most common approach to install Ambassador Edge Stack, especially in production environments, with our default, customizable manifest.
+
+### [<img class="helm-logo" src="../../../images/helm-navy.png"/> Install with Helm](#install-with-helm)
+
+Helm is a popular Kubernetes package manager. The Ambassador helm chart allows you to install Service Preview alongside the Ambassador Edge Stack.
+
+---
+
+## <img class="Ambassador's OpenSource Blackbird" src="../../../images/features-page-bird.svg"/> Install with Edgectl
 
 If you are a new user, or you are looking to start using Ambassador Edge Stack with Service Preview on a fresh installation, the `edgectl install` command will get you up and running in no time with a pre-configured Traffic Manager and Traffic Agent supported by automatic sidecar injection.
 
-Run the following command to let `edgectl` bootstrap your cluster with Ambassador and Service Preview
+### 1. Install the Traffic Manager and Ambassador Injector Alongside the Ambassador Edge Stack 
+
+The Traffic Manager is what is responsible for managing communications between your Kubernetes cluster and your local machine.
+
+Services in your cluster opt-in to using Service Preview by injecting the Traffic Agent sidecar. Service Preview includes an automatic sidecar injection feature which simplifies the process of injecting the Traffic Agent as sidecars to your services.
+
+Run the following command to let `edgectl` bootstrap your cluster with Ambassador, the Traffic Manager, and Ambassador Injector:
 
 ```sh
 $ edgectl install
 ```
 
-## Install with YAML
+### 2. Register and Install a License Key
```

Comment:
Not anymore! We should remove this step as 1.7 won't require a license to trial Service Preview.

Reply:
Wow how did I not understand that was what we were doing 🤦